infrastructure/kubernetes/common/monitoring/README.md

```diff
@@ -0,0 +1,73 @@
+# Kubernetes: monitoring
+
+This module contains Kubernetes deployments for monitoring the Mezo nodes.
+
+### Prerequisites
+
+- Infrastructure components for the `mezo-<environment>` GCP project created using the
+  corresponding Terraform module (make sure GKE version is >=1.29)
+- `gcloud` installed and authorized to access the `mezo-<environment>` GCP project
+- `kubectl` tool installed
+
+### Secrets
```

Comment:
Worth mentioning we need to create the `monitoring` namespace first. I see `namespace.yaml` but we create secrets before running Kustomize. Let's add a step saying that we need to do `kubectl create namespace monitoring` here. Without that, secrets don't work.

Reply:
good point. addressed in: f78c58814c41c13a91d802f00985f4869675fb76